teacher.rb

```diff
@@ -1,5 +1,5 @@
 class Teacher < Person
-  attattr_accessor :@specialization
+  attattr_accessor :specialization
```

Comment:
Well done on implementing the required changes. 👏🏾 

You have a typo here. The `attr_accessor` is misspelled with `attattr_accessor`. kindly fix that to prevent errors when utilizing the getter and setter for `: specialization`.

As a result, the following error occurs:

![image](https://user-images.githubusercontent.com/38088818/226102937-e38efaae-e855-4513-bf54-bd806c6f4964.png)